keyboards/albert_smart/albert_smart84/readme.md

```diff
@@ -0,0 +1,26 @@
+# Smart84
+
+![Smart84](https://i.imgur.com/D07mXuI.png)
```

Comment:
404's?

Reply:
Uh...could not loading the picture?

Reply:
the Chinese network cannot be direct,try to use VPN?

Reply:
Loads ok for me.